src/actions.ts

```diff
@@ -452,6 +491,60 @@ export function createActionsSystem(
       AvatarAttach.deleteFrom(entity)
     }
   }
+
+  // TELEPORT PLAYER
+  function handleTeleportPlayer(
+    _entity: Entity,
+    payload: ActionPayload<ActionType.TELEPORT_PLAYER>,
+  ) {
+    const { x, y } = payload
+    requestTeleport({
+      destination: `${x},${y}`,
+    })
+  }
+
+  // MOVE PLAYER
+  function handleMovePlayer(
+    _entity: Entity,
+    payload: ActionPayload<ActionType.MOVE_PLAYER>,
+  ) {
+    const { position, cameraTarget } = payload
+    const options = {
+      newRelativePosition: position,
+      cameraTarget,
+    }
+    // leaving this here for debugging purposes until we figure out why this does not work as intended
+    console.log('movePlayerTo', options)
+    movePlayerTo(options)
+  }
+
+  // PLAY DEFAULT EMOTE
+  function handlePlayDefaultEmote(
+    _entity: Entity,
+    payload: ActionPayload<ActionType.PLAY_DEFAULT_EMOTE>,
+  ) {
+    const { emote } = payload
+    void triggerEmote({ predefinedEmote: emote })
+  }
+
+  // PLAY CUSTOM EMOTE
+  function handlePlayCustomEmote(
+    _entity: Entity,
+    payload: ActionPayload<ActionType.PLAY_CUSTOM_EMOTE>,
+  ) {
+    const { src, loop } = payload
+    console.log('triggerSceneEmote', payload)
```

Comment:
is this `console.log` required for debugging too?

Reply:
i removed that one because we already know the issue is within the triggerSceneEmote